src/main/java/org/jitsi/jicofo/JitsiMeetConferenceImpl.java

```diff
@@ -3246,8 +3256,8 @@ public String toString()
     {
         return
             String.format(
-                    "JitsiMeetConferenceImpl[id=%s, name=%s]",
-                    id,
+                    "JitsiMeetConferenceImpl[gid=%s, name=%s]",
```

Comment:
Should the format be %d here now? I know Integer.toString() works, but semantically it seems odd